kythe/cxx/extractor/textproto/textproto_extractor.cc

```diff
@@ -47,6 +47,9 @@ ABSL_FLAG(std::string, proto_message, "",
 ABSL_FLAG(std::vector<std::string>, proto_files, {},
           "A comma-separated list of proto files needed to fully define "
           "the textproto's schema.");
+ABSL_FLAG(std::optional<std::string>, multiline_separator, std::nullopt,
+          "Delimitates each record within a recordio file. Presence of this"
+          "flag indicates this is a recordio formatted file.");
```

Comment:
"indicates this expects recordio formatted files

Reply:
Done.